src/dso_api/static/dso_api/dynamic_api/js/openapi_formatter.js

```diff
@@ -6,8 +6,11 @@
 // returns: HTML string
 
 const FORMATTER = (rawJson) => {
-
     let openApiEl = document.createElement('div');
+    var requestPath = window.location.pathname;
```

Comment:
```suggestion
    let requestPath = window.location.pathname;
```
graag consistentie met regel erboven